libsolidity/analysis/ContractLevelChecker.h

```diff
@@ -88,8 +88,8 @@ class ContractLevelChecker
 
 	/// Warns if the contract has a payable fallback, but no receive ether function.
 	void checkPayableFallbackWithoutReceive(ContractDefinition const& _contract);
-	/// Error if the contract requires too much storage
-	void checkStorageSize(ContractDefinition const& _contract);
+	/// Error if the contract requires too much storage or transient storage
+	void checkStorageSize(ContractDefinition const& _contract, VariableDeclaration::Location _location);
```

Comment:
At first I was going to add a different function, but then realized it's the same job, but different data location and error message.